lib/schnorr/vs_schnorr_wasm.js

```
@@ -984,30 +802,22 @@
+        let ptr2 = 0;
+        if (!isLikeNone(keyshare)) {
+            _assertClass(keyshare, Keyshare);
+            ptr2 = keyshare.__destroy_into_raw();
+        }
+        const ret = wasm.qcsession_new(ptr0, len0, ptr1, len1, ptr2);
+        if (ret[2]) {
+            throw takeFromExternrefTable0(ret[1]);
         }
+        this.__wbg_ptr = ret[0] >>> 0;
+        QcSessionFinalization.register(this, this.__wbg_ptr, this);
+        return this;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)